helpers/logs.go

```diff
@@ -63,3 +68,10 @@ func (il *IntegrationLogger) LogAuditEvent(msg string, obj runtime.Object, actio
 
 	log.Info(msg, keysAndValues...)
 }
+
+// WithApp returns a new logger with application.name and application.namespace key-values
+func (il IntegrationLogger) WithApp(app applicationapiv1alpha1.Application) IntegrationLogger {
+	log := il.Logger.WithValues("application.namespace", app.Namespace, "application.name", app.Name)
```

Comment:
WDYT about shortening to namespaced name, something like this (untested):
```
log := il.Logger.WithValues("application", fmt.Sprintf("%s/%s", application.Namespace, application.Name))
```

Reply:
not sure how hard would be searching in logs for all events in app namespace, but if we always need to search only application namespace/name together, it works for me

Reply:
let's make it shorter, I'll do PR. I cannot really think about usecase where we would need to lookup for something without namespace/name app combination

Reply:
added